app/controllers/password_resets_controller.rb

```diff
@@ -0,0 +1,57 @@
+class PasswordResetsController < ApplicationController
+  before_action :get_user, only: %i(edit update)
+  before_action :valid_user, only: %i(edit update)
+  before_action :check_expiration, only: %i(edit update)
```

Comment:
```suggestion
  before_action :get_user, :valid_user, :check_expiration, only: %i(edit update)
```
ngoài ra, ko nên bắt đầu tên method là set/get nhé, vì đôi lúc vô tình ghi đè method của rails á

Reply:
Dạ. ready ạ.

Reply:
Chỗ này bị lỗi quá 80 kí tự nên em có xuống dòng và tab như thế để tránh lỗi 80 kí tự và lỗi "Align the arguments of a method call if they span more than one line." ạ. ^^